plugins/github/api.ml

```diff
@@ -544,7 +544,7 @@ module Head_ref = Monitor(struct
           prefix
           name
           target {
-            ...on Commit {
```

Comment:
In the GraphQL language `...` should be used instead of the UTF-8 ellipsis `…` .